src/masternodes/mn_checks.cpp

```diff
@@ -377,6 +386,10 @@ class CCustomMetadataParseVisitor : public boost::static_visitor<Res>
         auto res = isPostEunosFork();
         return !res ? res : serialize(obj);
     }
+    Res operator()(CLoanSetCollateralTokenMessage& obj) const {
+        auto res = isPostFortCanningFork();
+        return serialize(obj);
```

Comment:
Will not fail under fork height, should be.

`return !res ? res : serialize(obj);`